files/images/teamspeak_viewer/badges/tim_irl_details.svg

```diff
@@ -0,0 +1,748 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 23.0.4, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" [
+	<!ENTITY ns_extend "http://ns.adobe.com/Extensibility/1.0/">
+	<!ENTITY ns_ai "http://ns.adobe.com/AdobeIllustrator/10.0/">
+	<!ENTITY ns_graphs "http://ns.adobe.com/Graphs/1.0/">
+	<!ENTITY ns_vars "http://ns.adobe.com/Variables/1.0/">
+	<!ENTITY ns_imrep "http://ns.adobe.com/ImageReplacement/1.0/">
+	<!ENTITY ns_sfw "http://ns.adobe.com/SaveForWeb/1.0/">
+	<!ENTITY ns_custom "http://ns.adobe.com/GenericCustomNamespace/1.0/">
+	<!ENTITY ns_adobe_xpath "http://ns.adobe.com/XPath/1.0/">
+]>
+<svg version="1.1" xmlns:x="&ns_extend;" xmlns:i="&ns_ai;" xmlns:graph="&ns_graphs;"
+	 xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 64 64"
+	 style="enable-background:new 0 0 64 64;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#1C80B7;}
+	.st1{fill:#234B7A;}
+	.st2{fill:#1C2538;}
+	.st3{fill:#FFFFFF;}
+	.st4{fill:#BFB882;}
+	.st5{fill:#D8CC84;}
+</style>
+<switch>
+	<foreignObject requiredExtensions="&ns_ai;" x="0" y="0" width="1" height="1">
+		<i:pgfRef  xlink:href="#adobe_illustrator_pgf">
+		</i:pgfRef>
+	</foreignObject>
+	<g i:extraneous="self">
+		<g id="Layer_1">
+			<path class="st0" d="M62.5,29.1L59,27.5c0.2-1.2,0-3.7,0-3.7C57.4,10.4,45.9,0,32,0C18.1,0,6.6,10.4,5,23.9c0,0-0.2,2.4,0,3.6
+				l-3.5,1.6C-3.3,37.8,5.1,46,5.1,46l4.2-1.5h45.5l4.2,1.5C58.9,46,67.3,37.8,62.5,29.1z"/>
+			<path class="st1" d="M31.9,4.6C31.9,4.6,31.9,4.6,31.9,4.6C31.9,4.6,31.9,4.6,31.9,4.6C31.9,4.6,31.9,4.6,31.9,4.6
+				C31.9,4.6,31.9,4.6,31.9,4.6C31.9,4.6,31.9,4.6,31.9,4.6C29.8,4.7-2.3,6.3,9.3,44.7l9.6,13.9L29.5,64h2.3h0.1h2.4l10.6-5.3
+				l9.6-14C66,6.5,34.3,4.7,31.9,4.6z"/>
+			<g id="Smirk_1_">
+				<path class="st0" d="M53.8,45c0,0-2.8,9.1-17.8,9c0,0-6.3-0.1-1.5,1.3s16.2,2.4,19.9-3.6c0,0,1.9-2.4,0.8-6.8
+					C55.2,44.8,54.7,43.4,53.8,45z"/>
+			</g>
+			<g id="Eyes_1_">
+				<path class="st3" d="M19.4,43.9c0,0-5.6-0.1-7-6.6s0.7-9.3,2.4-9.7s6.6,0.4,13.9,9.3C35.6,45.1,19.4,43.9,19.4,43.9z"/>
+				<path class="st3" d="M37.5,43.7c0,0-6-2.2-1-5.8c5-3.6,13.4-6.4,15.2-2.6c1.8,3.8-1.8,8.1-3.6,8.6C46.2,44.5,44,45.3,37.5,43.7z
+					"/>
+			</g>
+		</g>
+		<g id="Layer_2">
+		</g>
+	</g>
+</switch>
+<i:pgf  id="adobe_illustrator_pgf">
```

Comment:
this is not a correct svg

Reply:
Rechecked with original source. The file is untouched from original.